napalm/junos/utils/junos_views.yml

```diff
@@ -104,7 +104,7 @@ junos_lldp_table:
 junos_lldp_view:
   fields:
     hostname: lldp-remote-system-name
-    port: lldp-remote-port-description | lldp-remote-port-id
+    port: lldp-remote-port-id
```

Comment:
```suggestion
    port: lldp-remote-port-id | lldp-remote-port-description
```

Reply:
thanks for your point (I was not aware of this)